src/Menu/SelectMenu.test.jsx

```diff
@@ -15,7 +14,7 @@ const selectMenu = mount(
       <MenuItem iconBefore={Add}>A Menu Item With an Icon Before</MenuItem>
       <MenuItem iconAfter={Check}>A Menu Item With an Icon After </MenuItem>
       <MenuItem disabled>A Disabled Menu Item</MenuItem>
-      <MenuItem as={Hyperlink} href="https://en.wikipedia.org/wiki/Hyperlink">A Link Menu Item</MenuItem>
+      <MenuItem as="a" href="https://en.wikipedia.org/wiki/Hyperlink">A Link Menu Item</MenuItem>
```

Comment:
I think the error here is that `Hyperlink` expects `destination` prop and we pass `href` instead. If you replace `href` with `destination` it should fix the errors.

We shouldn't remove `Hyperlink`, it's a valid use-case for Paragon users.

Reply:
You are right, thanks